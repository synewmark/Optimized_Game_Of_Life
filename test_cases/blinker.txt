version https://git-lfs.github.com/spec/v1
oid sha256:ea5199bf63127d59e6bb7bbfcd9fe7aa1843a1a02e8e633262de301e509f8fe9
size 47
