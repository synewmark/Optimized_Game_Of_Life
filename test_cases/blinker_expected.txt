version https://git-lfs.github.com/spec/v1
oid sha256:add1227ad90d657ca9873c5310cd9ba51be3fb9620bb18123cb3abe2c25e9b15
size 35
